StoreTestWPF.Presentation/Controls/CakePreview.xaml.cs

```diff
@@ -0,0 +1,31 @@
+﻿using StoreTestWPF.DAL.Models;
+using StoreTestWPF.ViewModel.ViewModels;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace StoreTestWPF.Presentation.Controls
+{
+    /// <summary>
+    /// Interaction logic for CakePreview.xaml
+    /// </summary>
+    public partial class CakePreview : UserControl
+    {
+        public CakePreview()
+        {
+            InitializeComponent();
+        }
+
+        public CakeViewModel Cake
```

Comment:
As I remember we discussed this point. And we agreed to put Cake to the DataContext.

Reply:
Fixed